receiver/awsxrayreceiver/receiver.go

```diff
@@ -117,6 +117,7 @@ func (x *xrayReceiver) Shutdown(_ context.Context) error {
 func (x *xrayReceiver) start() {
 	incomingSegments := x.poller.SegmentsChan()
 	for seg := range incomingSegments {
+		x.obsrecv.StartTraceDataReceiveOp(seg.Ctx)
```

Comment:
This returns a context that should be used in the equivalent End.

Reply:
Edited in latest commit